include/SFML/Network/TcpListener.hpp

```diff
@@ -74,6 +74,10 @@ class SFML_NETWORK_API TcpListener : public Socket
     /// function is called, it will stop listening on the old
     /// port before starting to listen on the new port.
     ///
+    /// You can use the special value Socket::AnyPort to tell the
+    /// system to automatically pick an available port, and then
+    /// call getLocalPort to retrieve the chosen port.
```

Comment:
I'd have probably gone more with something like this:

```
/// When providing sf::Socket::AnyPort as port, the listener
/// will request an available port from the system.
/// The chosen port can be retrieved by calling getLocalPort().
```

Reply:
I basically copy-pasted the lines from UdpSocket.hpp: https://github.com/SFML/SFML/blob/master/include/SFML/Network/UdpSocket.hpp#L81-L83

Reply:
But I prefer your way. Do you want me do edit UdpSocket.hpp too to keep consistency?

Reply:
Oh I didn't know. Still kind of prefer my version. Either way, I think it makes sense to have the `sf::` prefix in both versions.

Reply:
Ok done. :)